assignments/objects.js

```diff
@@ -18,28 +18,71 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
+  "id":"1",
+  "name":"Mitzi",
+  "email":"mmelloy0@psu.edu",
+  "gender":"F"
+}
+
+const Kennan = {
+  "id":"2",
+  "name":"Kennan",
+  "email":"kdiben1@tinypic.com",
+  "gender":"M"
+}
+
+const Keven = {
+  "id":"3",
+  "name":"Keven",
+  "email":"kmummery2@wikimedia.org",
+  "gender":"M"
+}
+
+const Gannie = {
+  "id":"4",
+  "name":"Gannie",
+  "email":"gmartinson3@illinois.edu",
+  "gender":"M"
+}
+
+const Antonietta = {
+  "id":"5",
+  "name":"Antonietta",
+  "email":"adaine5@samsung.com",
+  "gender":"F"
+}
+
```

Comment:
You probably want to use let instead of const for all of your intern objects. Const is immutable, meaning that it is not changeable and you cannot reassign to the const variable. With a let variable, it is mutable and will allow for reassignment.